src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -166,14 +167,16 @@ private static FeedSource updateFeedSource(Request req, Response res) throws IOE
         }
         Persistence.feedSources.replace(feedSourceId, updatedFeedSource);
         // Notify feed- and project-subscribed users after successful save
-        NotifyUsersForSubscriptionJob.createNotification(
+        NotifyUsersForSubscriptionJob notifyUsersForFeedSubscriptionJob = new NotifyUsersForSubscriptionJob(
             "feed-updated",
             updatedFeedSource.id,
             String.format("Feed property updated for %s.", updatedFeedSource.name));
-        NotifyUsersForSubscriptionJob.createNotification(
+        notifyUsersForFeedSubscriptionJob.run();
```

Comment:
Use the light executor for this. See https://github.com/ibi-group/datatools-server/pull/348#discussion_r512337741 for reasoning.

Reply:
Scratch that, I think the previous way should be used. Please revert due to reasons list in https://github.com/ibi-group/datatools-server/pull/348#discussion_r512341045.

Reply:
Updated.